lib/validate-schemas.spec.js

```diff
@@ -49,33 +187,21 @@ const validateSchema = (filepath) =>
   });
 
 describe("validate schemas", () => {
-  it("should validate all schemas", (done) => {
-    const options = { absolute: true, ignore: "node_modules/**" };
-    glob("**/*schema.json", options, (err, files) => {
-      if (err) {
-        console.log(err);
+  it("should validate all schemas", async () => {
+    const options = { absolute: true, sync: true, ignore: "node_modules/**" };
+    const files = glob("types/**/*schema.json", options);
```

Comment:
Ah okay, we might want to add `"data-models/**/*schema.json"` eventually. But it's totally fine to do that after we refactored everything there.